src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
 from typing_extensions import Literal
 
 
+class Analytics(BaseModel):
+    number_matching: int = Field(
+        ...,
+        description='Number of workers matching hardware, platform, application, and custom requirements',
+        title='Number Matching',
+    )
+    number_assignable: int = Field(
+        ...,
+        description='Number of workers that are able to pick up tasks based on resource assignment mode',
+        title='Number Assignable',
+    )
+    number_accessible_assignable_and_matching: int = Field(
+        ...,
+        description='Number of workers matching hardware, platform, application, custom requirements, assignment and permissions',
+        title='Number Accessible Assignable And Matching',
+    )
+    number_accessible: int = Field(
+        ..., description='Number of workers that have permissions', title='Number Accessible'
+    )
+
+
+class AnalyzeHpcResources(BaseModel):
+    num_cores_per_node: Optional[int] = Field(None, title='Num Cores Per Node')
+    num_gpus_per_node: Optional[int] = Field(None, title='Num Gpus Per Node')
```

Comment:
```suggestion
    num_gpus_per_node: Optional[int] = Field(None, title='Num Gpus Per Node')
```
Do the descriptions that appear in "Field" show in the documentation? If so, they should be written out in plain English without using abbreviations. The should also conclude with periods.